catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/resource/download/ReliableResourceDownloader.java

```diff
@@ -494,18 +494,16 @@ public void run() {
               downloadIdentifier);
         }
       }
-    } catch (IOException | InterruptedException e) {
+    } catch (IOException e) {
       LOGGER.info("Unable to store product file {}", filePath, e);
-      Thread.currentThread().interrupt();
 
       downloadState.setDownloadState(DownloadState.FAILED);
```

Comment:
Noticed that I didn't remove both of these calls here. I've fixed this for the next commit.